source/manual/alerts/content-performance-manager-app-healthcheck-not-ok.html.md

```diff
@@ -35,8 +35,19 @@ This means the [the ETL master process][1] that runs daily has failed to collect
 To fix this problem run the [following rake task][2]:
 
 ```bash
-rake etl:repopulateviews[2018-01-01 2018-01-01]
+rake etl:repopulate_views[2018-01-01 2018-01-01]
+```
+
+## ETL :: no searches for yesterday
+
+This means the [the ETL master process][1] that runs daily has failed to collect `number of searches` metrics from Google Analytics. The issue may originate from the ELT processor responsible for collecting Internal Searches.
```

Comment:
Typo

Reply:
Fixed in https://github.com/alphagov/govuk-developer-docs/pull/1524/commits/07309a700cb0f72e12ac53ba4b706ee0c7e89185